### PR TITLE
[DEV APPROVED] IE8 homepage SVG fallbacks

### PR DESCRIPTION
**Pull request by johnplayer1982**
Adding homepage SVG fallbacks for older browsers.  Includes Newsletter icon and arrow icon in striped banner:

* Before:
screen shot 2015-11-02 at 15 31 38

* After:
screen shot 2015-11-02 at 15 31 42

### CODE REVIEW

**Comment by benbarnett** [comment]
Can you just check some of the engine pages (e.g. mortgage calculator/pension calculator). Just want to check the sprite sheet having a new height doesn't affect anything downstream.

**Comment by johnplayer1982** [comment]
Good point @benbarnett I've checked the mortage and pensions calculators locally and the icons are unaffected by the increased height, It would be ok as long as we've added to the bottom or right and havent used %'s for background position using the main sprite sheet, which I have checked for

File: app/views/shared/svg/_icon_sprite.html.erb [hunk 1]
```diff
@@ -27,7 +27,14 @@
...
   </symbol>
   <symbol id="svg-icon--youtube" viewBox="0 0 17 11.957">
-    <path d="M6.745 8.184V3.407l4.593 2.396-4.593 2.38zM16.83 2.58s-.166-1.17-.676-1.687c-.646-.677-1.37-.68-1.703-.72C12.073 0 8.505 0 8.505 0h-.008S4.928 0 2.55.172C2.216.212 1.49.215.845.892.336 1.408.17 2.58.17 2.58S0 3.956 0 5.33v1.29c0 1.377.17 2.752.17 2.752s.166 1.172.676 1.688c.646.677 1.496.656 1.875.727 1.36.13 5.78.17 5.78.17s3.573-.004 5.952-.176c.332-.04 1.057-.042 1.703-.72.51-.515.676-1.687.676-1.687S17 7.997 17 6.62V5.33c0-1.374-.17-2.75-.17-2.75z"/></svg>
+    <path d="M6.745 8.184V3.407l4.593 2.396-4.593 2.38zM16.83 2.58s-.166-1.17-.676-1.687c-.646-.677-1.37-.68-1.703-.72C12.073 0 8.505 0 8.505 0h-.008S4.928 0 2.55.172C2.216.212 1.49.215.845.892.336 1.408.17 2.58.17 2.58S0 3.956 0 5.33v1.29c0 1.377.17 2.752.17 2.752s.166 1.172.676 1.688c.646.677 1.496.656 1.875.727 1.36.13 5.78.17 5.78.17s3.573-.004 5.952-.176c.332-.04 1.057-.042 1.703-.72.51-.515.676-1.687.676-1.687S17 7.997 17 6.62V5.33c0-1.374-.17-2.75-.17-2.75z"/>
   </symbol>
+
+  <!-- Homepage stripe banner icon -->
```
Comment (benbarnett) [inline app/views/shared/svg/_icon_sprite.html.erb]: I think the class name describes this, so no need for the comment here

File: app/assets/stylesheets/components/common/_icon.scss [hunk 2]
```diff
@@ -357,3 +356,21 @@
   height: 25px;
   width: 20px;
 }
+
+// Stripe banner - Homepage
```
Comment (benbarnett) [inline app/assets/stylesheets/components/common/_icon.scss]: Same here re: the comment, can remove

**Comment by benbarnett** [comment]
Cool then.

:+1: aside from small points

**Comment by rnalexander** [comment]
:+1: :shipit:

**Pull request merged**